spec/unit/reservation_list_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'spec_helper'
+
+describe "Reservation index list", type: :feature, js: true do
+
+  let(:user) { FactoryGirl.create(:user) }
+
+  it "should get reservation lists from proper day" do
+    FactoryGirl.create(:reservation, from: '10', to: '11')
+    FactoryGirl.create(:reservation, from: '12', to: '13')
+    FactoryGirl.create(:reservation, from: '14', to: '15')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.